Blueye.Protocol.Protobuf.csproj

```diff
@@ -5,20 +5,20 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <LangVersion>8.0</LangVersion>
     <PackageId>Blueye.Protocol.Protobuf</PackageId>
-    <VersionPrefix>3.8.0</VersionPrefix>
+    <VersionPrefix>3.10.0</VersionPrefix>
```

Comment:
Jumping over 3.9.0 is intentional?

Reply:
Yes. I try to "pair" the .NET version of the package with version of the Blueye App that will go-out with this version.